internals/cli/cmd_run.go

```diff
@@ -31,16 +31,16 @@ import (
 	"github.com/canonical/pebble/internals/systemd"
 )
 
-const cmdRunSummary = "Run the pebble environment"
+const cmdRunSummary = "Run the <display name> environment"
 const cmdRunDescription = `
 The run command starts pebble and runs the configured environment.
 
 Additional arguments may be provided to the service command with the --args option, which
-must be terminated with ";" unless there are no further Pebble options.  These arguments
+must be terminated with ";" unless there are no further program options.  These arguments
 are appended to the end of the service command, and replace any default arguments defined
 in the service plan. For example:
 
-    $ pebble run --args myservice --port 8080 \; --hold
+    <program name> run --args myservice --port 8080 \; --hold
```

Comment:
Lost a `$` here. That's okay if it's fixing the style that is already in use everywhere else. If it's mixed, let's please leave that alone and do it in a separate PR addressing the style alone, since there will likely be other strings not related to the program name that will need changing too.